### PR TITLE
 Issue/#146 Update README.md

### PR DESCRIPTION
**Pull request by rishab-goswami**
- [ ] The pull request is complete according to the following criteria:
  - [ ] Acceptance criteria have been met
  - [ ] The documentation is kept up-to-date
  - [ ] Comprehensive tests (if applicable) have been generated and all pass.
  - [ ] The pull request describes the changes that have been made, and enough information is present in the description for any developer to understand what has changed
  - [ ] Commits have been squashed (or will be on merge).
  - [ ] The branch name is descriptive and follows the pull request title format : {issue/bug...}/(Issue Number) - Name of issue. E.g bug/30-Fix-Project
  - [ ] The pull request title is of the following format : {issue/bug...}/(Issue Number) - Name of issue. E.g bug/30-Fix-Project
  - [ ] The description uses [github syntax](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) to link to the issue. E,g Resolves se701g2/Doto#{Number}
  - [ ] At least two reviewers assigned. One of which must be the assigner of the issue.
  - [ ] If there are merge conflicts, run git rebase as opposed to git merge with master.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,12 @@ We are using npm for this particular project (https://www.npmjs.com/get-npm).
 
 Please check the front end [readme](https://github.com/se701g2/Doto/blob/master/doto-frontend/README.md) for more information on running the front end of the code.
 
+### Link to the website 
+doto.azurewebsites.net
```
Comment (daltonlim) [inline README.md]: Nit: Could you have it link directly to the website

Comment (rishab-goswami) [reply README.md]: apologies, I forgot it doesn't automatically do it. Done now

Comment (daltonlim) [reply README.md]: Wow that was fast. I think the link requires http:// prefix for it to work. It currently links to a github page.

File: README.md [hunk 2]
```diff
@@ -21,10 +21,10 @@ We are using npm for this particular project (https://www.npmjs.com/get-npm).
 Please check the front end [readme](https://github.com/se701g2/Doto/blob/master/doto-frontend/README.md) for more information on running the front end of the code.
 
 ### Link to the website 
-doto.azurewebsites.net
+[doto.azurewebsites.net](doto.azurewebsites.net)
```
Comment (daltonlim) [inline README.md]: Wow that was fast. I think the link requires http:// prefix for it to work. It currently links to a github page.

Comment (rishab-goswami) [reply README.md]: Yeah, I just added it

**Pull request merged**